crates/ruff_python_ast/src/comparable.rs

```diff
@@ -495,6 +495,36 @@ impl<'a> From<&'a ast::ExceptHandler> for ComparableExceptHandler<'a> {
     }
 }
 
+#[derive(Debug, PartialEq, Eq, Hash)]
+pub enum ComparableFStringPart<'a> {
+    Literal(&'a str),
+    FormattedValue(FormattedValue<'a>),
+}
+
+#[derive(Debug, PartialEq, Eq, Hash)]
+pub struct FormattedValue<'a> {
+    expression: ComparableExpr<'a>,
+    debug_text: Option<&'a ast::DebugText>,
+    pub conversion: ast::ConversionFlag,
+    pub format_spec: Vec<ComparableFStringPart<'a>>,
+}
```

Comment:
Can you provide the reason for keeping these (`conversion`, `format_spec`) 2 fields public?

Reply:
copy/paste fail i think